### ISSUE TITLE
Blazor Server App

### ISSUE THREAD

**Issue by ltemimi** [dotnet/AspNetCore.Docs#21527]
Hi

Can a blazor server app be hosted as a windows service?
If yes any steps on how to do it would b appreciated

regards
laz

**Comment by guardrex** [comment]
Hello @ltemimi ... In theory, *yes*, but there's no official guidance for that scenario at this time outside of perhaps hosting Blazor Server using the [Windows Service guidance in the main doc set](https://docs.microsoft.com/aspnet/core/host-and-deploy/windows-service). WASM may have some **_gotchas_** 😈 that I haven't looked into yet.

This issue is a dup of ...

* https://github.com/dotnet/AspNetCore.Docs/issues/19294
* https://github.com/dotnet/AspNetCore.Docs/issues/20159

Given that this isn't a high priority for coverage because it isn't a common hosting scenario for Blazor, I recommend that if you run into trouble while waiting on formal guidance to consult community-based coverage (e.g., blog posts) and community support. We recommend the usual support channels ...

* [Stack Overflow (tag: `blazor`)](https://stackoverflow.com/questions/tagged/blazor)
* [ASP.NET Core Slack Team](http://tattoocoder.com/aspnet-slack-sign-up/)
* [Blazor Gitter](https://gitter.im/aspnet/Blazor)

I'll close this as a duplicate of the existing issues. I have a number of high priority doc overhauls to complete prior to .NET 6 landing, but I expect that we'll (probably) have coverage for this scenario sometime this year.
